README.md

```diff
@@ -1,39 +1,22 @@
-# talon_one
-Use the Talon.One API to integrate with your application and to
-manage applications and campaigns:
-
-- Use the operations in the [Integration API section](#integration-api)
-are used to integrate with our platform
-- Use the operation in the [Management API section](#management-api) to
-manage applications and campaigns.
-
-## Determining the base URL of the endpoints
-
-The API is available at the same hostname as your Campaign Manager deployment.
-For example, if you access the Campaign Manager at `https://yourbaseurl.talon.one/`,
-the URL for the [updateCustomerSessionV2](https://docs.talon.one/integration-api#operation/updateCustomerSessionV2) endpoint
-is `https://yourbaseurl.talon.one/v2/customer_sessions/{Id}`
-
+# Talon.One Python SDK
 
 This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
-- API version: 
-- Package version: 6.1.1
+- Package version: 6.7.0
```

Comment:
I guess this is a demonstration leftover?

Reply:
Good catch, will fix that in the template.